src/ai/backend/client/namegenerator.py

```diff
@@ -0,0 +1,360 @@
+    "dubinsky",
+    "easley",
+    "edison",
+    "einstein",
+    "elbakyan",
+    "elgamal",
+    "elion",
+    "ellis",
+    "engelbart",
+    "euclid",
+    "euler",
+    "faraday",
+    "feistel",
+    "fermat",
+    "fermi",
+    "feynman",
+    "franklin",
+    "gagarin",
+    "galileo",
+    "galois",
+    "ganguly",
+    "gates",
+    "gauss",
+    "germain",
+    "goldberg",
+    "goldstine",
+    "goldwasser",
+    "golick",
+    "goodall",
+    "gould",
+    "greider",
+    "grothendieck",
+    "haibt",
+    "hamilton",
+    "haslett",
+    "hawking",
+    "hellman",
+    "heisenberg",
+    "hermann",
+    "herschel",
+    "hertz",
+    "heyrovsky",
+    "hodgkin",
+    "hofstadter",
+    "hoover",
+    "hopper",
+    "hugle",
+    "hypatia",
+    "ishizaka",
+    "jackson",
+    "jang",
+    "jemison",
+    "jennings",
+    "jepsen",
+    "johnson",
+    "joliot",
+    "jones",
+    "kalam",
+    "kapitsa",
+    "kare",
+    "keldysh",
+    "keller",
+    "kepler",
+    "khayyam",
+    "khorana",
+    "kilby",
+    "kirch",
+    "knuth",
+    "kowalevski",
+    "lalande",
+    "lamarr",
+    "lamport",
+    "leakey",
+    "leavitt",
+    "lederberg",
+    "lehmann",
+    "lewin",
+    "lichterman",
+    "liskov",
+    "lovelace",
+    "lumiere",
+    "mahavira",
+    "margulis",
+    "matsumoto",
+    "maxwell",
+    "mayer",
+    "mccarthy",
+    "mcclintock",
+    "mclaren",
+    "mclean",
+    "mcnulty",
+    "mendel",
+    "mendeleev",
+    "meitner",
+    "meninsky",
+    "merkle",
+    "mestorf",
+    "mirzakhani",
+    "montalcini",
+    "moore",
+    "morse",
+    "murdock",
+    "moser",
+    "napier",
+    "nash",
+    "neumann",
+    "newton",
+    "nightingale",
+    "nobel",
+    "noether",
+    "northcutt",
+    "noyce",
+    "panini",
+    "pare",
+    "pascal",
+    "pasteur",
+    "payne",
+    "perlman",
+    "pike",
+    "poincare",
+    "poitras",
+    "proskuriakova",
+    "ptolemy",
+    "raman",
+    "ramanujan",
+    "ride",
+    "ritchie",
+    "rhodes",
+    "robinson",
+    "roentgen",
+    "rosalind",
+    "rubin",
+    "saha",
+    "sammet",
+    "sanderson",
+    "satoshi",
+    "shamir",
+    "shannon",
+    "shaw",
+    "shirley",
+    "shockley",
+    "shtern",
+    "sinoussi",
+    "snyder",
+    "solomon",
+    "spence",
+    "stonebraker",
+    "sutherland",
+    "swanson",
+    "swartz",
+    "swirles",
+    "taussig",
+    "tereshkova",
+    "tesla",
+    "tharp",
+    "thompson",
+    "torvalds",
+    "tu",
+    "turing",
+    "varahamihira",
+    "vaughan",
+    "villani",
+    "visvesvaraya",
+    "volhard",
+    "wescoff",
+    "wilbur",
+    "wiles",
+    "williams",
+    "williamson",
+    "wilson",
+    "wing",
+    "wozniak",
+    "wright",
+    "wu",
+    "yalow",
+    "yonath",
+    "zhukovsky",
+]
+
+
+def get_random_name() -> str:
+    return f"{random.choice(left)}_{random.choice(right)}"
+
+
+def duplicated_name(name: str) -> str:
+    return f"{name}{random.randrange(0, 10)}"
```

Comment:
the name of the function is ambiguous because what this function does is appending random integer for duplicated Session names.
Let's rename it into a more intuitive name like `append_random_int` etc